src/main/java/uk/gov/companieshouse/siccode/api/search/SicCodeService.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.siccode.api.search;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.query.TextCriteria;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SicCodeService {
+
+    @Autowired
+    private CombinedSicActivitiesRepository combinedSicActivitiesRepository;
```

Comment:
Consider using constructor injection instead of field injection:
https://reflectoring.io/constructor-injection/

Reply:
I like that website too

Reply:
Changed